vayesta/core/types/wf/ccsdtq.py

```diff
@@ -14,13 +14,15 @@ def CCSDTQ_WaveFunction(mo, *args, **kwargs):
 
 class RCCSDTQ_WaveFunction(wf_types.WaveFunction):
 
-    def __init__(self, mo, t1, t2, t3, t4_abab, t4_abaa):
+    def __init__(self, mo, t1, t2, t3, t4):
         super().__init__(mo)
         self.t1 = t1
         self.t2 = t2
         self.t3 = t3
-        self.t4_abab = t4_abab
-        self.t4_abaa = t4_abaa
+        self.t4 = t4
+        if not (isinstance(t4, tuple) and len(t4) == 2):
+            raise ValueError('''t4 definition in RCCSDTQ wfn requires 
+                    tuple of (abaa, abab) spin signatures''')
```

Comment:
This message will contain a newline and many spaces, due to the the multiline string.
Note also that I allow myself up to 120 columns, so I wouldn't break here.
As a side note, I try to follow the following convention:
* `"` for text or sentences, i.e. everything that will be read by a human (error messages, logging, docstrings).
* `'` for keys, options, or enums: (i.e. `my_dictionary['c2aa']` or `my_function(solver='CCSD')`.

Reply:
Good to know. Changed.